.github/workflows/ci.yml

```diff
@@ -115,6 +115,8 @@ jobs:
         cmake -S . -B out -G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
     - name: build
       run: cmake --build out
+    - name: test lit
+      run: python out/bin/binaryen-lit test/lit/parse-error.wast
```

Comment:
Is this just testing that the lit wrapper works at all? Is this meant to be in the final PR?

Reply:
Yeah I thought it might be worth testing directly.. just just via check.py.